3.random_forest/random-forest-pipeline.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+#
+# Process the data using methods centered around random forest classifiers
+set -e
+
```

Comment:
```suggestion

# First, convert all notebooks to scripts
jupyter nbconvert --to=script \
  --FilesWriter.build_directory=scripts/nbconverted \
  *.ipynb
```

This line will convert the notebook to a python script for easier commenting

Reply:
But that script is not used or run anywhere. Is it the idea no notebooks are present in the final product?

Reply:
notebooks are one of the major final products!

All files in `scripts/nbconverted` are just so its easy to view the actual python code _without_ the figures and syntax of `.ipynb` files. Its tough to pick out the actual python (or R) code in the `.ipynb` files.